cuda/preconditioner/jacobi_kernels.cu

```diff
@@ -65,8 +65,6 @@ constexpr int default_num_warps = 32;
 // current GPUs have at most 84 SMs)
 constexpr int default_grid_size = 32 * 32 * 128;
 
-constexpr int default_block_size = 512;
```

Comment:
Any reason this was removed ?

Reply:
it's unused because those kernels are moved to unified kernel